### ISSUE TITLE
Getting error while running on colab

### ISSUE THREAD

**Issue by SirRatul** [sagorbrur/bendeep#3]
I am following the documentation of Machine Translation and trying to run the code on colab. But I am getting the following error

`Error: 

Explanation: 
Sentence contains out of vocabulary word. That means sentence has a word which not exist in trained data
/usr/local/lib/python3.7/dist-packages/torch/serialization.py:656: SourceChangeWarning: source code of class 'torch.nn.modules.sparse.Embedding' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.7/dist-packages/torch/serialization.py:656: SourceChangeWarning: source code of class 'torch.nn.modules.rnn.GRU' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.7/dist-packages/torch/serialization.py:656: SourceChangeWarning: source code of class 'torch.nn.modules.linear.Linear' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.7/dist-packages/torch/serialization.py:656: SourceChangeWarning: source code of class 'torch.nn.modules.dropout.Dropout' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)`

FYI, I have successfully run the "Sentiment Analysis" code on colab and got proper output.

**Comment by sagorbrur** [comment]
Hi @SirRatul,
bendeep translation module has some issues with vocabulary. I think I need to re-implement this module with better codes.
After re-implementation, I will get back to you soon. Meanwhile, you can try to use [this](https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html) code and try to run from scratch. 
regards

**Comment by SirRatul** [comment]
Hi @sagorbrur,
Thank you very much for your reply.  I'll try to run that code and wait for the re-implementation. Best of luck.
